content/applications/hr/payroll.rst

```diff
@@ -0,0 +1,13 @@
+:nosearch:
+:show-content:
+:hide-page-toc:
+:show-toc:
```

Comment:
If there is no content in this page, then you only need `:nosearch:` and certainly not `:show-content:`

```suggestion
:nosearch:
```

You can learn more about [metadata in the contributing doc](https://www.odoo.com/documentation/master/contributing/documentation/rst_cheat_sheet.html#document-metadata).

I you intend to add basic/overview content on the app page, which I encourage greatly, then you'll add the appropriate metadata in that commit, but not here, since there is no content yet.